index.js

```diff
@@ -165,7 +169,11 @@ var Omegle = function () {
 	this.reloadReCAPTCHA = function () {
 		if (challengeLink) evalCaptcha(challengeLink);
 	};
-	this.updateServer = function () {
+	this.updateServer = function (server) {
+		if(server){	//in case user wants to do it manually
+			url='http://'+server;
+			return false;
+		}
```

Comment:
I don't think this is necessary as there's no point in manually setting the server. So please remove it for now, we can probably add something like this later if a use case for this comes up.